Dockerfile.common.in

```diff
@@ -4,6 +4,13 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 USER root
 
+# Install Composer 1.9.1
+# https://github.com/greenpeace/planet4-docker/blob/master/config.default#L42
+RUN php -r "copy('https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer', 'composer-setup.php');" && \
```

Comment:
If composer was already installed, we can probably use `composer self-update --1` instead of running the installer again. Seems a bit safer too since there might be other things needed for the downgrade that the installer doesn't do?

Reply:
Seems to [work](https://app.circleci.com/pipelines/github/greenpeace/planet4-circleci-unit-tests/93/workflows/2d714641-c4e5-4e8b-baaf-8da94ddcd750/jobs/118) :+1: